src/main/java/org/amahi/anywhere/adapter/RecentFilesAdapter.java

```diff
@@ -54,8 +56,19 @@ public RecentFilesViewHolder onCreateViewHolder(ViewGroup parent, int viewType)
 
     @Override
     public void onBindViewHolder(@NonNull RecentFilesViewHolder holder, int position) {
-        RecentFile file = recentFiles.get(position);
-        setUpViewHolder(file, holder);
+        if (showShimmer) {
+            holder.shimmerFrameLayout.startShimmer();
+        } else {
+            holder.shimmerFrameLayout.stopShimmer();
+            holder.shimmerFrameLayout.setShimmer(null);
+            holder.fileIconView.setBackground(null);
+            holder.fileTextView.setBackground(null);
+            holder.moreOptions.setBackground(null);
+            holder.fileSize.setBackground(null);
+            holder.fileLastVisited.setBackground(null);
```

Comment:
Please create a new function dedicated to stopping shimmer for better code readability.

Reply:
Also, why do we have to individually set background of all items to null if we are setting frame layout's background to null?

Reply:
I can't see where we are setting the frame layout's background null.

Reply:
I have refactored the code in the latest commit please see it